spec/components/provider_interface/activity_log_event_component_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::ActivityLogEventComponent do
+  def component_for(event)
+    described_class.new(activity_log_event: event)
+  end
+
+  def with_event(trait)
+    event = create(:application_choice_audit, trait)
+    user = event.user.display_name
+    candidate = event.auditable.application_form.full_name
+    yield event, user, candidate
+  end
+
+  describe '#event_description' do
+    examples = {
+      awaiting_provider_decision: '<candidate> submitted an application',
+      withdrawn: '<candidate> withdrew their application',
+      with_rejection: '<user> rejected <candidate>’s application',
+      with_rejection_by_default: '<candidate>’s application was rejected automatically',
+      with_rejection_by_default_and_feedback: '<user> sent feedback to <candidate>',
+      with_offer: '<user> made an offer to <candidate>',
+      with_modified_offer: '<user> made an offer to <candidate>',
+      with_accepted_offer: '<candidate> accepted an offer',
+      with_declined_offer: '<candidate> declined an offer',
+      with_declined_by_default_offer: '<candidate>’s offer was declined automatically',
+      with_withdrawn_offer: '<user> withdrew <candidate>’s offer',
+      with_recruited: '<user> recruited <candidate>',
```

Comment:
```suggestion
      with_recruited: '<candidate> met all offer conditions',
```
We don't use the term 'recruited' within the service'. Adam and I think this a more accurate description of what has happened.